src/Admin/SiteHealth.php

```diff
@@ -102,6 +102,10 @@ public function register() {
 		add_filter( 'site_status_test_result', [ $this, 'modify_test_result' ] );
 		add_filter( 'site_status_test_php_modules', [ $this, 'add_extensions' ] );
 
+		if ( version_compare( get_bloginfo( 'version' ), '6.1', '>=' ) && has_filter( 'amp_page_cache_good_response_time_threshold' ) ) {
+			add_filter( 'site_status_good_response_time_threshold', 'get_good_response_time_threshold' );
```

Comment:
This is supposed to be a method, right?

```suggestion
			add_filter( 'site_status_good_response_time_threshold', [ $this, 'get_good_response_time_threshold' ] );
```

Reply:
Looks like these lines need a test since it didn't cause any errors.

Reply:
Yes, but why it didn't cause any errors 🤔

Reply:
Humm, because the place where it is hooked is not called. This is because cache site health tests only run in the production environment.

https://github.com/WordPress/wordpress-develop/blob/ce342f1a33051dcafdb47132f8e07d0597f49124/src/wp-admin/includes/class-wp-site-health.php#L2828